en/docs/assets/versions.json

```diff
@@ -1,13 +1,21 @@
 {
-  "next": "5.8.0",
-  "current": "5.7.0",
+  "next": "5.9.0",
```

Comment:
Why next also `5.9.0` ?

Reply:
We don't know what version number we will release next for sure right?